docs/community/glossary.md

```diff
@@ -0,0 +1,15 @@
+| dock widget        | A widget that can be attached to the main napari viewer at a specified position.                                                                                                                                                                                                                                                            |
+| hook               | Hook is the place within napari where we “promise” to call functions created by external developers and installed by the user.                                                                                                                                                                                                              |
+| hook specification | A function signature that defines the API that a plugin developer must adhere to when writing the function that we promise to call somewhere in the napari codebase.                                                                                                                                                                        |
+| keybinding         | The key or key combination that causes some functions to occur.                                                                                                                                                                                                                                                                             |
+| labels             | The labels layer is an image where pixel values are integers. Each unique integer represents a unique object, feature or classification.                                                                                                                                                                                                    |
+| layer              | Layers are the viewable objects that can be added to napari viewer. Currently napari supports the following layers: image, labels, points, shapes, surface, tracks, vectors.                                                                                                                                                                |
+| mousebinding       | The specific mouse click that causes some functions to occur.                                                                                                                                                                                                                                                                               |
+| plugin             | A napari plugin is a specialized Python package which extends napari's functionality through additional file i/o support, image analysis, or other utilities. Plugin developers specify this functionality through a combination of Python entry points and a specialized manifest file. Plugins for napari can be found at napari-hub.org. |
```

Comment:
I like this!
* Split the first sentence - put a full stop after "functionality" and make the second half a standalone sentence (examples of extended functionality)
* Link to the napari plugins section of the docs

Reply:
Not sure about the mention of the manifest just yet since that's still very much in flight at the moment?